doc/vagrant.md

```diff
@@ -0,0 +1,25 @@
+## Testing `kube-spawn` with [Vagrant](https://www.vagrantup.com/)
+
+The provided Vagrantfile is used to test `kube-spawn` on various Linux distributions.
+
+```
+$ vagrant up
+$ vagrant ssh
+$ ./build.sh    # sets up environment, runs build and up/init command
+```
+
+To run them manually:
```

Comment:
This duplicates what already can be found in `## Quickstart`. Not needed in my opinion (and one more thing to keep up-to-date), let's drop it?